src/main/features/claim/paths.ts

```diff
@@ -5,12 +5,16 @@ export class Paths {
   static readonly taskListPage = new RoutablePath('/claim/task-list')
   static readonly resolvingThisDisputerPage = new RoutablePath('/claim/resolving-this-dispute')
   static readonly completingClaimPage = new RoutablePath('/claim/completing-claim')
-  static readonly claimantYourDetailsPage = new RoutablePath('/claim/your-details')
+  static readonly claimantIndividualDetailsPage = new RoutablePath('/claim/claimant-individual-details')
```

Comment:
I do not see much reason for having separate path for each claimant type. That causes duplicates in route classes and to some extent view files as well.

CHANGE needed